tree/20_cfe_basics/log_rudder.cf

```diff
@@ -16,9 +16,19 @@
 #
 #####################################################################################
 
-# @name Log for Rudder
-# @description Logging output for Rudder reports
+# @name Log for Rudder with mode
+# @description Logging different verbosity outputs for Rudder reports
+# Supported mode are:
+#  log_repaired
+#  log_warn
+#  log_info
+#  log_debug
+#  log_trace
 #
+# If the parameter mode is not in the list above, it will report in a classical manner
+# based on the reporting context and the defined classes.
```

Comment:
I'm really confused on how it should work: it has a class prefix, that it won't use to get the _repaired, _success, _failure, so the message will also be not be customized

why not simply having a 
bundle agent log_rudder_mode(mode, message, class_parameter, class_prefix) ?
signature will only expose what's relevant, and make it easier to use.